Sources/SwiftLocation/Frequency.swift

```diff
@@ -105,8 +105,8 @@ public func <(lhs: Frequency, rhs: Frequency) -> Bool {
 		return true
 	case (.deferredUntil(let d1,_,_), .deferredUntil(let d2,_,_)):
 		return d1 < d2
-	case (.significant, .significant):
-		return true
+	case (.significant(let backgroundUpdates1), .significant):
+		return backgroundUpdates1
```

Comment:
This should always prioritize the Frequency setting which has backgroundUpdates enabled in the scenario where the user has applied two .significant Frequency trackers.